var/spack/repos/builtin/packages/py-pillow/package.py

```diff
@@ -32,7 +32,8 @@ class PyPillow(Package):
     capabilities."""
 
     homepage = "https://python-pillow.org/"
-    url      = "https://pypi.python.org/packages/source/P/Pillow/Pillow-3.0.0.tar.gz"
+    url = "https://pypi.python.org/packages/source" + \
+        "/P/Pillow/Pillow-3.0.0.tar.gz"
```

Comment:
This can be left as one line. We put special handling in run-flake8-tests to ignore long lines like URLs because they're less readable if you have to split them up.